README.rst

```diff
@@ -115,6 +115,7 @@ Then add a event url handler in your `urls.py`::
     ]
 
 SESEventWebhookView handles bounce, complaint, send, delivery, open and click events.
+It is also capable of auto confirming subscriptions, it handles `SubscriptionConfirmation`, `UnsubscribeConfirmation` notifications.
```

Comment:
The naming here is a bit inconsistent. Let's do `UnsubscriptionConfirmation` maybe? Or `SubscribeConfirmation` if you prefer - so the two words are more aligned.

Reply:
Understood, fine as is then! Thanks.